lib/EditableTextWrapper/styles.scss

```diff
@@ -8,6 +8,7 @@
 .editable-text__wrapper {
   display: flex;
   margin: 0 0 0.25em;
+  width: 100%;
```

Comment:
Make sure this doesn't cause any issues on the app!

There's a few places where you've got the 100% width set like this instead of a helper class. Just curious to know where you draw the line of using a helper vs using css? 

Personally I think helpers are more of a external (i.e. not used in ui) tool, since in my head they're mostly overrides that I want to use on UI components that im using. If this used the `h-width-100` helper, I'm not sure I would be able to override it with a `h-width-80` helper elsewhere.

Reply:
It's a good question.

I've overused them here I think. I'll stick to how we've used them before and add some fixed styles. I think because I added them I overly used them 😄